src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.MultiAgent.cs

```diff
@@ -546,9 +549,38 @@ private void FinishRequest(EasyRequest completedOperation, CURLcode messageResul
                 completedOperation.Cleanup();
             }
 
+            private static unsafe void CurlDebugFunction(IntPtr curl, Interop.Http.CurlInfoType type, IntPtr data, ulong size, IntPtr context)
```

Comment:
I'm no compiler, so I may be wrong, but does this method need to be marked 'unsafe'?